test/mocha/tests.js

```diff
@@ -0,0 +1,34 @@
+/*global describe, it, xit*/
+function assert(expr, msg) {
+    if (!expr) {
+        throw new Error(msg || "failed");
+    }
+}
+
+describe("group a", function () {
+    it("foo", function () {
+        assert(5 === 5);
+        assert(1 / 0 === Infinity);
+    });
+
+    it("bar goes wrong", function () {
+        assert(5 === "5");
+    });
+});
+
+describe("group b", function () {
+    it("baz", function () {
+        assert(true);
+    });
+    xit("skipped test", function () {
```

Comment:
Maybe its wiser to have the more conventional `.skip()` test too? `xdescribe()` and `xit()` maybe deprecated in the future.
http://tjholowaychuk.tumblr.com/post/29996505898/mocha-140